SPEC.md

```diff
@@ -201,7 +201,7 @@ The baseline remediation points value is 50,000, which is the time it takes to f
 
 ### Locations
 
-Locations refer to ranges of a source code file. A Location contains a `path`, a source range, (expressed as `lines` or `positions`), and an optional array of `other_locations`. Here's an example location:
+Locations refer to ranges of a source code file. A Location contains a `path`, a source range, (expressed as `lines` or `positions`). Here's an example location:
```

Comment:
Since we've fixed this by dropping the reference to `other_location`, we now have some grammar to clean up here. (I think it should be "contains a `path` and a source range (", not "contains a `path`, a source range, ("